test/cpp/tensorexpr/test_loopnest.cpp

```diff
@@ -14,6 +14,7 @@
 #include <torch/csrc/jit/tensorexpr/loopnest.h>
 #include <torch/csrc/jit/tensorexpr/tensor.h>
 #include <torch/csrc/jit/testing/file_check.h>
+#include "torch/csrc/jit/tensorexpr/ir_simplifier.h"
```

Comment:
Nit: please use <> and sort includes.

Reply:
I thought clang-format was supposed to do that for me :-(